### PR TITLE
Rsv005 reservation payment confirm Component

### PR DESCRIPTION
**Pull request by andress20**
**Creation of function component for confirm reservation payment.**

- [x] Structure separate by styled, logic and component
- [x] New component add in route component
- [x] Add deposit information for specific restaurant

**modified files**
`src/App.js` => add new pages in the routes
`ItemShoppingCart.js` => Get deposit for selected restaurant and add button to go confirmation phase
`Restaurants.js` => add deposit information.
`reservationReducer.js` => add inital state
`Restaurant.js` => Add deposit to reducer
`ReservationForm` => minimal fix wrong variable name

**new files**
`ReservationConfirm`
`components/Reservation/ReservationConfirmPayment.js` => render page
`components/Reservation/ReservationStyles.js `

reservationpayment

### CODE REVIEW

**Review by SergioJaramilloL — DISMISSED** [review]

File: src/components/ItemShoppingCart.js [hunk 1]
```diff
@@ -26,6 +28,16 @@ const TextStrongTotal = styled.td`
...
 
 export function ListShoppingCart ( { deposit } ) {
+  
+  const data = useSelector(
+    ({reservationReducer: {
+      ...state 
+    }}) => {
+    return { ...state }
+  })
+  console.log('reservation confirm:', data)
```
Comment (simonhoyos) [inline src/components/ItemShoppingCart.js]: Ojo con los console.logs

**Review by simonhoyos — CHANGES_REQUESTED** [review]

File: src/components/ItemShoppingCart.js [hunk 2]
```diff
@@ -36,14 +48,18 @@ export function ListShoppingCart ( { deposit } ) {
...
-            <td>{deposit}</td>
+            <td>{data.deposit}</td>
           </tr>
           <tr>
             <TextStrongTotal>Total</TextStrongTotal>
-            <td>{deposit}</td>
+            <td>{data.deposit}</td>
           </tr>
           </LabelTable>
+        <Link to={{
```
Comment (carosalazar28) [inline src/components/ItemShoppingCart.js]: Estás envolviendo un button en un link? no es mejor dejar solo uno, que sea el Link

**Review by carosalazar28 — CHANGES_REQUESTED** [review]

File: src/components/ItemShoppingCart.js (hunk 1 above)
Comment (andress20) [reply src/components/ItemShoppingCart.js]: listo, tendre mas cuidado.

File: src/components/ItemShoppingCart.js (hunk 2 above)
Comment (andress20) [reply src/components/ItemShoppingCart.js]: Listo @carosalazar28, ya quedo implementado como sugeriste. gracias!

**Review by carosalazar28 — APPROVED** [review]

**Review by SergioJaramilloL — APPROVED** [review]

**Review by simonhoyos — APPROVED** [review]

**Pull request merged**